repos/spack_repo/builtin/packages/py_metatrain/package.py

```diff
@@ -0,0 +1,51 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.python import PythonPackage
+
+from spack.package import *
+
+
+class PyMetatrain(PythonPackage):
+    """Train, fine-tune, and manipulate machine learning models for atomistic systems"""
+
+    homepage = "https://docs.metatensor.org/metatrain/latest/index.html#"
+    pypi = "metatrain/metatrain-2025.10.tar.gz"
+
+    maintainers("RMeli", "luthaf", "HaoZeke")
+
+    license("BSD-3-Clause", checked_by="RMeli")
+
+    version("2025.11", sha256="4c1a78feb359885f29d659f97ccf0e0676892ba1aa120cf58d0c7a0442161255")
+    version("2025.10", sha256="70ca3d422dbaf8eca14f05f81299497faf52134bcf478b29484e891ba0881549")
+
+    variant("soap_bpnn", default=False, description="Enable SOAP-BPNN architecture")
+
+    # pyproject.toml [project]
+    depends_on("python@3.9:", type=("build", "run"), when="@2025.10:")
```

Comment:
I think we can remove this, Python 3.9 is EOL, so unless it is specifically supported by spack I would get rid of it